lib/index.js

```diff
@@ -48,30 +48,13 @@ SatisMeter.prototype.loaded = function() {
  * @param {Identify} identify
  */
 
-SatisMeter.prototype.identify = function(identify) {
-  var traits = identify.traits();
-  traits.token = this.options.apiKey || this.options.token;
-  traits.user = {
-    id: identify.userId()
-  };
-
-  if (identify.name()) {
-    traits.user.name = identify.name();
-  }
-  if (identify.email()) {
-    traits.user.email = identify.email();
-  }
-  if (identify.created()) {
-    traits.user.signUpDate = identify.created().toISOString();
```

Comment:
@hankim813 This is no longer needed as it's handled directly in our library (satismeter.js).